src/main/java/com/tasks/taskapi/utils/TaskExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+package com.tasks.taskapi.utils;
+
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.data.crossstore.ChangeSetPersister.NotFoundException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import com.tasks.taskapi.exception.FailedUpdateException;
+
+@ControllerAdvice
+public class TaskExceptionHandler extends ResponseEntityExceptionHandler {
+    
+    @ExceptionHandler(value = {RuntimeException.class})
+    protected ResponseEntity<Object> handleException(
+        RuntimeException exception, WebRequest request) {
+            String bodyOfResponse = "i made a mistake sorry :(";
+            return handleExceptionInternal(exception, bodyOfResponse,
```

Comment:
maybe try to be more consistent here and return response entity in each method or use handleExceptionInternal() in each method